src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -292,26 +290,24 @@ private static Win32 InternetShortcutProgram(string path)
             string[] lines = System.IO.File.ReadAllLines(path);
             string appName = string.Empty;
             string iconPath = string.Empty;
+            string urlPath = string.Empty;
             string scheme = string.Empty;
             bool validApp = false;
 
-            const string steamScheme = "steam";
+            Regex urlSchemes = new Regex(@"^steam:\/\/rungameid\/|^com\.epicgames\.launcher:\/\/apps\/");

```

Comment:
Can you please add back the case where hostname for steam apps is run? Thanks.

Reply:
@alekhyareddy28 Thanks for pointing out this. I will modify this.